IdentityAccess/src/Kreta/IdentityAccess/Infrastructure/Ui/Http/Symfony/public/images

```diff
@@ -0,0 +1 @@
+../../../Uploads/images/
```

Comment:
`/public/uploads` its better IMO. I would leave `/images` free in case we need to put some default assets. This will require to change the url parameter discussed above